### ISSUE TITLE
DESIGN 65

### ISSUE THREAD

**Issue by nikhilreddydev** [rishikeshreddyakkireddy/designtodevelopment#137]
Develop this design using your preferred web technologies. Try to approximate it up to 80%.
Don't worry about colors much, but try to make the UI neat and clean.

# DESIGN

![d65](https://user-images.githubusercontent.com/77785504/136666807-39d6dfc6-0642-40af-a1fa-28a6379eaa97.png)

# Resources

**Assets that are needed, are available in folder Design65**  

1: Color Palette generator : https://coolors.co/
2: Icons : https://fontawesome.com/
3: Fonts : https://fonts.google.com/
4: Images : https://unsplash.com/

**Comment by rishikeshreddyakkireddy** [comment]
start contributing :)

**Comment by nikhilreddydev** [comment]
@Evavic44 No, no. we only need the static version of it.

But, we will appreciate. if they can manage to implement that..

**Comment by Ardent10** [comment]
@nikhilreddydev It is done but having some difficulty related to opening a PR. I know it's silly still I want to ask you 
Here is the description of what I did: 
- created a branch patch1
- Completed the task 
- committed
- then upon pushing the I used git push --set-upstream patch1

This created a new branch in my forked repo "patch1". Now when I try to open a PR it shows "Able to merge" upon comparing the forks from mine to yours, but no options to create a pull request.

Any suggestions?
